modules/ROOT/pages/_partials/endpoint-https-send.adoc

```diff
@@ -1,6 +1,6 @@
 == HTTP(S) Send Configuration
 
-Use these settings when sending with the secure HTTP(S) protocols.
+Use these settings when sending with HTTP and the secure HTTPS protocol.
 This includes endpoints for sending transmissions to your partner and endpoints that are targets at your organization.
```

Comment:
See suggestion above.
Carefully evaluate to be sure that the suggestion does not impair your original intended meaning.

Reply:
Rewritten with your edits in mind.